src/scripts/activate-user.ts

```diff
@@ -21,15 +36,56 @@ export const activateUser = (
         O.fold(
           () => TE.left(`User [${userId}] does not exist. Cannot activate.`),
           // eslint-disable-next-line @typescript-eslint/prefer-readonly-parameter-types
-          () => service.activateUser(userId)
+          (user) => TE.right(user)
         )
       )
-    ),
-    TE.chainFirstIOK((user) =>
-      Console.info(`Activated [${user.id}] [${user.email}].`)
     )
   );
 
+export const checkUserStatusPriorToActivation = (
```

Comment:
**issue:** document exported members.

Reply:
added tsdoc to exported members in commit: expire-password-and-get-temporary-password